docs/getting_started.md

```diff
@@ -15,6 +15,9 @@
 5. For UI testing follow the instructions in [openshift console](https://github.com/openshift/console#dependencies) repository.
    After clone of repository and setting all dependencies please create configuration file with RUN section with `openshift_console_path`
    parameter pointing to your local clone of openshift console repo. See example in `conf/ocsci/ui-testing.yaml`.
+6. Install the dependencies for scipy
+
+sudo  yum install atlas-devel openssl-devel libffi-devel gcc
```

Comment:
can you align this text properly?

Reply:
While we're at it this should be in a code block since it's a full command

Reply:
@vasukulkarni : I assume you meant the code-block alignment. I incorporated @clacroix12 recommendation. Let me know if it addresses yours as well.